Skip rows without timestamp when parsing reported errors

hansmi opened the pull request:
The table with errors encountered by the heat pump controller may contain empty rows, denoted by `----`.

hansmi commented:
@jeroentorrekens, can you please test whether the proposed changes resolve issue #5 reported by you?

jeroentorrekens commented:
I'm afraid the issue still exists.

```
2022/03/31 22:19:49 Scrape failed: collection via LuxWS protocol failed: parsing time "----" as "02.01.06 15:04:05": cannot parse "----" as "02"
```

The code change makes sense though: remove all leading and trailing dashes and then check if you have an empty string left. I'm also a bit clueless on why it's not working. I'll try to do some more debugging tomorrow and see if I can come up with something.

hansmi commented:
@jeroentorrekens, I've updated the exporter program with a `-verbose` flag and rebased this pull request. When set the flag will give you the communication messages sent between the exporter and the controller:

```console
$ ./luxws-exporter -verbose -controller.address=192.0.2.1:8214 -controller.language=en
[…]
2022/04/02 10:00:00 Sending message of type 1: "GET;0x7a1234"
2022/04/02 10:00:00 Received message of type 1: "<Content><item id='0x7babcd'>[…]
```

This should help with your investigation.

jeroentorrekens commented:
@hansmi this is embarrassing... 15 years of being a linux sysadmin, but I was testing (/usr/local/bin/)luxws-exporter instead of ./luxws-exporter. Your fix worked, I was just testing the wrong binary. All data is exported successfully.
Thanks!

hansmi commented:
@jeroentorrekens, thank you for testing again. I figured it must be something like that, but then again I could've made a mistake in the code.
